src/components/ProductList/ProductList.js

```diff
@@ -4,15 +4,25 @@ import Product from '../Product';
 const ProductList = props => {
   let products;
 
-  props.productList.length > 0
-  ? products = props.productList.map(product => <Product {...product} key={product.id} />)
-  : products = <h2>No results.</h2>
+    if (!props.productList) {
+      return null;
+    } else {
 
-    return(
-      <ul className="product-list">
-        {products}
-      </ul>
-    );
+    props.productList.length > 0
+    ? products = props.productList.map(product => <Product
+      key={product.id}
+      {...product}
+      addToWishList={props.addToWishList}
+      removeFromWishList={props.removeFromWishList}
+      wishList={props.wishList} />)
+    : products = <h2>No results.</h2>
+
```

Comment:
I think another way you could write this is:
`products = props.productList.length > 0 ? (1st case) : <h2>No results.</h2>`
It's slightly less code.